components/brave_rewards/core/contribution/contribution.cc

```diff
@@ -84,6 +86,21 @@ ledger::mojom::ContributionStep ConvertResultIntoContributionStep(
 namespace ledger {
 namespace contribution {
 
+Contribution::ContributionRequest::ContributionRequest(
+    std::string publisher_id,
+    double amount,
+    bool set_monthly,
+    base::OnceCallback<void(bool)> callback)
+    : publisher_id(std::move(publisher_id)),
+      amount(amount),
+      set_monthly(set_monthly),
+      callback(std::move(callback)) {}
+
+Contribution::ContributionRequest::ContributionRequest(
```

Comment:
We should default the move-assignment operator as well.

Reply:
Done.